monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ComponentControllerV1.java

```diff
@@ -474,4 +501,30 @@ static void setPropertyType(ComponentDescriptor<?> descriptor, ConfiguredPropert
             propInfo.setSchema(visitor.finalSchema());
         }
     }
+
+    public enum OutputStyle {
+
+        TABULAR,
+        MAP;
+
+        public static final String HTTP_PARAM_TABULAR = "tabular";
+        public static final String HTTP_PARAM_MAP = "map";
+        public static final String HTTP_PARAM_DOCUMENT = "document";
```

Comment:
public static fields inside an enum ... That'll be pretty confusing when one tries to figure out what is what in this class. What I mean is that when a user in his IDE types "OutputStyle (dot)" then he will see a list of 5 elements: The two enums and the three public static fields. Somehow this conceptually feels wrong to me...

Do we need to declare them like this? I would be OK with just an inline string declaration below...